src/main/scala/mesosphere/marathon/util/RichEither.scala

```diff
@@ -0,0 +1,39 @@
+package mesosphere.marathon
+package util
+
+import scala.collection.generic.CanBuild
+
+object RichEither {
+  // Remove for Scala 2.12
```

Comment:
no idea what this comment means!

Reply:
It means that since Right-biased Either is the default for Scala 2.12, this code will be unnecessary.

IE it's a polyfill :)